back-end/models/league_tables.model.js

```diff
@@ -1,5 +1,28 @@
 const db = require("../db/connection");
 
+exports.selectCompletedFixtures = () => {
+    return db
+        .query(
+            `
+      SELECT
+        team1.team_name AS team1_name,
+        team2.team_name AS team2_name,
+        team1_score,
+        team2_score,
+        team1.team_division AS division
+      FROM fixtures
+      JOIN teams AS team1
+        ON fixtures.team1_id = team1.team_id
+      JOIN teams AS team2
+        ON fixtures.team2_id = team2.team_id
+      WHERE fixtures.match_status = 'completed' 
```

Comment:
Say a team called XYZ has never completed any match, shouldn't the query also capture it and subsequently display it in the league table?

Reply:
Indeed it should (and does). This is achieved by a second query call in the controller (reusing the teams model). The util function creates a zero score league table row for every team and then iterates through the fixtures to add the necessary valued.

Reply:
If you make a get request in insomnia (or just in your browser as it is only a GET request) you should see teams who haven't played any games in the response